velodyne_driver/include/velodyne_driver/driver.h

```diff
@@ -74,6 +74,7 @@ class VelodyneDriver
     int    npackets;                 // number of packets to collect
     double rpm;                      // device rotation rate (RPMs)
     int cut_angle;                   // cutting angle in 1/100°
+    int timestamp_angle;            // configured phase lock angle
```

Comment:
Could you please include a description somewhere of what this actually means/does? It's definitely unclear to me.